src/main/java/io/openliberty/tools/ant/SpringBootUtilTask.java

```diff
@@ -79,7 +79,7 @@ public void execute() {
             }
         } else {
             throw new BuildException("An invocation of the " + command_name
-                    + " command failed. The command is not available on the targeted Liberty runtime.");
+                    + " command failed because it requires a Spring Boot feature to be configured in the server.xml file");
```

Comment:
`"The " + command_name + " command could not be invoked because it does not exist. Add the Spring Boot feature to the Liberty server configuration and try again."`

Reply:
Changed the error message. Thanks!